examples/tor_socks.rs

```diff
@@ -0,0 +1,24 @@
+#![deny(warnings)]
+
+// This is using the `tokio` runtime. You'll need the following dependency:
+//
+// `tokio = { version = "0.2", features = ["macros"] }`
+#[tokio::main]
+async fn main() -> Result<(), reqwest::Error> {
+    // Make sure you are running tor and this is your socks port
+    let proxy = reqwest::Proxy::all("socks5://127.0.0.1:9050").expect("tor proxy should be there");
```

Comment:
I'm not used to using TOR, is this the most common address?

Reply:
Yes, it should be. The Tor Browser uses 9150, I think, but that's not usually what other applications connect over.